src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/dto/OffenderSearch.kt

```diff
@@ -1,33 +1,22 @@
 package uk.gov.justice.digital.hmpps.incentivesapi.dto
-
+import uk.gov.justice.digital.hmpps.incentivesapi.dto.prisonapi.PrisonerAlert
+import uk.gov.justice.digital.hmpps.incentivesapi.service.PrisonerInfoForNextReviewDate
 import java.time.LocalDate
 
 data class OffenderSearchPrisoner(
-  val prisonerNumber: String,
-  val bookingId: Long,
+  override val bookingId: Long,
+  override val prisonerNumber: String,
+  override val dateOfBirth: LocalDate,
+  override val receptionDate: LocalDate,
   val firstName: String,
   val middleNames: String? = null,
   val lastName: String,
-  val status: String,
-  val inOutStatus: String,
```

Comment:
I got rid of some fields we don't actually use:
- `status`
- `inOutStatus`
- `prisonName`
- `cellLocation`
- `locationDescription`
- (we probably don't use `prisonId` but Igor suggested could be useful for debugging)

No point keeping these around unused fields and more often than not unused fields can cause deserialisation issues (when types are not 100% correct)